faculty_cli/version.py

```diff
@@ -17,16 +17,15 @@
 import platform
 from pkg_resources import get_distribution, DistributionNotFound
 
-
 try:
-    __version__ = get_distribution("sml").version
+    __version__ = get_distribution("faculty-cli").version
 except DistributionNotFound:
     # package is not installed
     pass
 
 
 def user_agent():
     """Return user agent string."""
-    return "sml/{} (Python {}, {})".format(
+    return "faculty/{} (Python {}, {})".format(
```

Comment:
Now we've split the CLI out from the library, shall we make the user agent `faculty-cli`?

Reply:
I was wondering about this. I don't have strong feelings but I don't think `faculty-cli` would introduce any confusion and would only help clarify

Reply:
> should we also set a user agent in the faculty library itself?

It might be helpful in showing what versions are still in use if we need to deprecate certain features in future.